apps/test-quadrature64cpr/src/main.cpp

```diff
@@ -0,0 +1,60 @@
+#include "Logger.h"
+#include "Quadrature64CPR.h"
+
+// first pin is yellow wire
+// second pin is white wire
+constexpr Encoder::Quadrature64CPR::Config config = {PA_5, PA_6, NC, 0};
+Encoder::Quadrature64CPR encoder(config);
+
+// Synchronous testing
+void Updater();
+void Reader();
+Timer timer;
+
+constexpr auto PERIOD            = 500ms;
+constexpr bool USE_BLOCKING_TEST = true;
+
+// Print helper function
+void print(const std::string &str);
+Mutex print_mutex;
+
+int main() {
+  Thread updater_thread(osPriorityNormal), reader_thread(osPriorityNormal);
+
+  updater_thread.start(&Updater);
+  reader_thread.start(&Reader);
+
+  updater_thread.join();
+  reader_thread.join();
+}
+
+void Updater() {
+  while (true) {
+    if constexpr (USE_BLOCKING_TEST) {
```

Comment:
You can take out this USE_BLOCKING_TEST condition, aeat-6012 has it because it has a blocking and async version of the driver, but this quadrature driver doesn't

Reply:
done